src/AddressInput/index.d.ts

```diff
@@ -9,16 +9,17 @@ export interface AddressInputProps {
   className?: string;
   clearButton?: boolean;
   initialValue?: string;
-  onSelect?: (value: DropdownLayoutValueOption) => void;
-  debounceDuration?: number;
+  value?: string;
+  disabled?: boolean;
+  onSelect?: (option: DropdownLayoutValueOption) => void;
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
   options?: DropdownLayoutOption[];
   onClear?: () => void;
-  status?: 'loading' | 'error' | 'warning';
+  status?: 'loading';
   roundInput?: boolean;
-  optionsLayout?: 'single-line' | 'double-line';
-  showOptionsIcons?: boolean;
   size?: 'small' | 'medium' | 'large';
+  placeholder?: string;
+  noResultsText?: string | React.ReactNode;
```

Comment:
String is a sub  type of React.ReactNode
```suggestion
  noResultsText?: React.ReactNode;
```